socorro/external/es/super_search_fields.py

```diff
@@ -380,9 +380,6 @@ def add_field_to_properties(properties, namespaces, field):
                 '_all': {
                     'enabled': False,
                 },
-                '_source': {
-                    'compress': True,
-                },
```

Comment:
This was a leftover from my first tests with ES 5.x, where that feature has been removed. It still exists in 2.x, so I'm going to put it back. Thanks for spotting that!